src/main/java/game/gameinternal/instance/Instance.java

```diff
@@ -0,0 +1,46 @@
+package game.gameinternal.instance;
+
+import org.json.JSONObject;
+
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+
+/**
+ * Created by Installed on 17.04.2016.
+ */
+@SuppressWarnings("PublicField")
+public class Instance {
+
+    @SuppressWarnings("FieldCanBeLocal")
+    private final ScriptEngine scriptEngine;
+    private final String name;
+    public final InstanceRunner instanceRunner;
+    public final InstanceOperator instanceOperator;
+    public InstanceChecker instanceChecker;
+
+    public Instance(String name, String path, String pathToOutput, JSONObject entry) {
+        this.name = name;
+        scriptEngine = new ScriptEngineManager().getEngineByName("nashorn");
+        instanceOperator = new InstanceOperator(scriptEngine);
+        instanceRunner = new InstanceRunner(name, path, pathToOutput, scriptEngine, entry);
```

Comment:
![My minds when saw it](https://s-media-cache-ak0.pinimg.com/236x/ab/90/c2/ab90c26e07eb1796b00767f052e99fab.jpg)